integration_tests/commands/resp/getwatch_test.go

```diff
@@ -101,10 +105,13 @@ func TestGETWATCHWithSDK(t *testing.T) {
 		watch := subscriber.client.WatchConn(context.Background())
 		subscribers[i].watch = watch
 		assert.True(t, watch != nil)
-		firstMsg, err := watch.Watch(context.Background(), "GET", getWatchKey)
+
+		uuid := uuid.New().String()
```

Comment:
I don't think we need to send the uuid on the user side, the SDK already creates one for us inside the Watch() method.
```suggestion
```